### ISSUE TITLE
Function accessibility issue

### ISSUE THREAD

**Issue by shavitush** [alliedmodders/sourcepawn#163]
Error output from SourceMod:

```
L 12/10/2017 - 02:02:55: SourceMod error session started
L 12/10/2017 - 02:02:55: Info (map "de_nuke") (file "errors_20171210.log")
L 12/10/2017 - 02:02:55: [SM] Exception reported: Fatal error creating dynamic native!
L 12/10/2017 - 02:02:55: [SM] Blaming: testcase.smx
L 12/10/2017 - 02:02:55: [SM] Call stack trace:
L 12/10/2017 - 02:02:55: [SM]   [0] CreateNative
L 12/10/2017 - 02:02:55: [SM]   [1] Line 6, E:\scripting\testcase.sp::AskPluginLoad2
L 12/10/2017 - 02:02:55: [SM]   [2] Line 21, E:\scripting\testcase.sp::GetClientButtons
```

Minimal test case:

```sp
#include <sourcemod>
#include <sdkhooks>

public APLRes AskPluginLoad2(Handle myself, bool late, char[] error, int err_max)
{
	CreateNative("FakeNative_1", Native_Fake1);
}

public int Native_Fake1(Handle handler, int numParams)
{
	
}

public void OnClientPutInServer(int client)
{
	SDKHook(client, SDKHook_PostThinkPost, PostThinkPost);
}

void PostThinkPost(int client)
{
	GetClientButtons(client);
}
```

Changing `PostThinkPost`'s accessibility to `public` gets rid of the error.
However, the stack trace outputted from SourceMod is terribly wrong and doesn't relate to the actual error in the code.

**Comment by peace-maker** [comment]
Which compiler version did you use?
This looks like it's fixed already. Might have been #123 or #125 

```
SourcePawn Compiler 1.9.0.5921 (spider)

; .3048.PostThinkPost
; 7 instruction(s)
; starts at code address 0xbe8
---
0xbec: break                           
0xbf0: break                           
0xbf4: push.s 0xc                       ; client
0xbfc: push.c 0x1                       ; 1
0xc04: call 0xb68                       ; AskPluginLoad2
0xc0c: zero.pri                        
0xc10: retn

SourcePawn Compiler 1.9.0.6225
; .3336.PostThinkPost
; 7 instruction(s)
; starts at code address 0xd08
---
0xd0c: break                           
0xd10: break                           
0xd14: push.s 0xc                       ; client
0xd1c: push.c 0x1                       ; 1
0xd24: call 0xb68                       ; GetClientButtons
0xd2c: zero.pri                        
0xd30: retn
```

**Comment by shavitush** [comment]
True. I've used the compiler bundled into SourceMod 1.8 at the time of posting this issue. Seems solved with SP 1.9!
